toolchain/check/handle_literal.cpp

```diff
@@ -2,7 +2,10 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#include <cwchar>
+
```

Comment:
```suggestion
```

Am I missing a use, or was this added due to some intermediate change?

Reply:
Yeah, more vs code automatic wrong `#include` insertion.